pre-docs/inventory_version2.md

```diff
@@ -75,8 +75,12 @@ currently Bolt only ships builtin lookup plugins. The plugins are:
 
 #### PuppetDB
 
-The PuppetDB plugin takes a `query` field, which is either a string containing
-a PQL query or an array containing a PuppetDB AST format query.
+The PuppetDB plugin takes a `query` field, which is either a string
+containing a
+[PQL](https://puppet.com/docs/puppetdb/latest/api/query/v4/pql.html) query or
+an array containing a [PuppetDB
+AST](https://puppet.com/docs/puppetdb/latest/api/query/v4/ast.html) format
+query.
 
```

Comment:
This is used to determine which targets should be included in the group. If no fact lookup is specified the certname for each item in the query result will used as the uri for the new target.